projects/fusion-e2e/entities/tests/chart/chartLabelIcon.pw.ts

```diff
@@ -0,0 +1,50 @@
+import {expect, Locator, test} from '@playwright/test';
+import {ChartPage} from '../../pages/chart-page';
+import {chartLabelWithIconStoryId} from './consts';
+import {CART_LABELS_ICONS_MOCK} from '@ironsource/fusion-ui/components/chart-labels/v4/chart-labels-v4.stories.mock';
+import {toRgba} from '../utils';
+
+const ALPHA_VALUE = 100;
```

Comment:
it seems you only use ALPHA_VALUE in one test so why declare it in a scope that is common for all tests?

Reply:
Fixed